api/order/orderRouter.js

```diff
@@ -0,0 +1,102 @@
+const express = require('express');
+const authRequired = require('../middleware/authRequired');
+const Orders = require('./orderModel');
+const router = express.Router();
+
+router.get('/', authRequired, function (req, res) {
+  Orders.findAll()
+    .then((orders) => {
+      res.status(200).json(orders);
+    })
+    .catch((err) => {
+      console.log(err);
+      res.status(500).json({ message: err.message });
+    });
+});
+
+router.get('/:id', authRequired, function (req, res) {
+  const id = String(req.params.id);
+  Orders.findById(id)
+    .then((order) => {
+      if (order) {
+        res.status(200).json(order);
+      } else {
+        res.status(404).json({ error: 'OrderNotFound' });
+      }
+    })
+    .catch((err) => {
+      res.status(500).json({ error: err.message });
+    });
+});
+
+router.post('/', authRequired, async (req, res) => {
+  const newOrder = req.body;
+  if (newOrder) {
+    const id = newOrder.id || 0; //0 needs to be replaced with a string
```

Comment:
0 needs to be replaced here before merging.
Can you look into `cuid` and the `uuid` libraries and use either one of those to replace this `0`?